main.go

```diff
@@ -106,6 +107,7 @@ func main() {
 		log.Fatalf(err.Error())
 	}
 
+	//not sure this should be in a tomb
```

Comment:
We don't want to be blocked by this. The main thread does signal handling and such below.